Exercises/1-power.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
 
 // Define function power(exp, n), the same as Math.pow(n, exp)
-// but with reverse order of argumants.
-const power = null;
+// but with reverse order of arguments.
+const power = (exp, n) => Math.pow(n, exp);
 
 // Implement function `square(n)`, which returns square of its argument.
 // The function may or may not reuse function `power`.
-const square = null;
+const square = (n) => power(2, n);
 
 // Implement function `cube(n)` using partial application
-// The function should return cube of argumant (to the power of three).
-const cube = null;
+// The function should return cube of argument (to the power of three).
+const cube = (n) => Math.pow.bind(null, n)(3);
```

Comment:
You can bind `power` function

Reply:
I can't
Error: Function cube expected to be binded Math.pow